Error fetching url [hentai-cafe]

* Ripme version: 1.7.70
* Java version: <!-- (output of `java -version`) --> 1.8.0_161
* Operating system: <!-- (if Windows, output of `ver` or `winver`) --> Linux Ubuntu
* Exact URL you were trying to rip when the problem occurred: https://hentai.cafe/chobipero-club/

I have tried on 2 different machines (a vps and my desktop) and got the same error.

## Expected Behavior

Ripme downloads all the images in the gallery

## Actual Behavior

```
Retrieving https://hentai.cafe/chobipero-club/
Error while loading https://hentai.cafe/manga/read/chobipero-club/en/0/1/<br />

org.jsoup.HttpStatusException: HTTP error fetching URL. Status=400, URL=https://hentai.cafe/manga/read/chobipero-club/en/0/1/<br%20/>
        at org.jsoup.helper.HttpConnection$Response.execute(HttpConnection.java:459)
        at org.jsoup.helper.HttpConnection$Response.execute(HttpConnection.java:434)
        at org.jsoup.helper.HttpConnection.execute(HttpConnection.java:181)
        at com.rarchives.ripme.utils.Http.response(Http.java:130)
        at com.rarchives.ripme.utils.Http.get(Http.java:110)
        at com.rarchives.ripme.ripper.rippers.HentaiCafeRipper.getFirstPage(HentaiCafeRipper.java:47)
        at com.rarchives.ripme.ripper.AbstractHTMLRipper.rip(AbstractHTMLRipper.java:86)
        at com.rarchives.ripme.App.rip(App.java:99)
        at com.rarchives.ripme.App.ripURL(App.java:265)
        at com.rarchives.ripme.App.handleArguments(App.java:248)
        at com.rarchives.ripme.App.main(App.java:74)
[!] Error while ripping URL https://hentai.cafe/chobipero-club/
java.io.IOException: Failed to load https://hentai.cafe/manga/read/chobipero-club/en/0/1/<br />
 after 1 attempts
        at com.rarchives.ripme.utils.Http.response(Http.java:137)
        at com.rarchives.ripme.utils.Http.get(Http.java:110)
        at com.rarchives.ripme.ripper.rippers.HentaiCafeRipper.getFirstPage(HentaiCafeRipper.java:47)
        at com.rarchives.ripme.ripper.AbstractHTMLRipper.rip(AbstractHTMLRipper.java:86)
        at com.rarchives.ripme.App.rip(App.java:99)
        at com.rarchives.ripme.App.ripURL(App.java:265)
        at com.rarchives.ripme.App.handleArguments(App.java:248)
        at com.rarchives.ripme.App.main(App.java:74)
Caused by: org.jsoup.HttpStatusException: HTTP error fetching URL. Status=400, URL=https://hentai.cafe/manga/read/chobipero-club/en/0/1/<br%20/>
        at org.jsoup.helper.HttpConnection$Response.execute(HttpConnection.java:459)
        at org.jsoup.helper.HttpConnection$Response.execute(HttpConnection.java:434)
        at org.jsoup.helper.HttpConnection.execute(HttpConnection.java:181)
        at com.rarchives.ripme.utils.Http.response(Http.java:130)
        ... 7 more
```

--- a/src/main/java/com/rarchives/ripme/ripper/rippers/HentaiCafeRipper.java
+++ b/src/main/java/com/rarchives/ripme/ripper/rippers/HentaiCafeRipper.java
@@ -44,7 +44,12 @@ public class HentaiCafeRipper extends AbstractHTMLRipper {
         public Document getFirstPage() throws IOException {
             // "url" is an instance field of the superclass
             Document tempDoc = Http.url(url).get();
-            return Http.url(tempDoc.select("div.last > p > a.x-btn").attr("href")).get();
+            String firstPageUrl = tempDoc.select("div.last > p > a.x-btn").attr("href");
+            // workaround for https://github.com/RipMeApp/ripme/issues/1083
+            if (firstPageUrl.contains("<br />")) {
+                firstPageUrl = firstPageUrl.replaceAll("<br />", "");
+            }
+            return Http.url(firstPageUrl).get();
         }
 
         @Override

--- a/src/test/java/com/rarchives/ripme/tst/ripper/rippers/HentaicafeRipperTest.java
+++ b/src/test/java/com/rarchives/ripme/tst/ripper/rippers/HentaicafeRipperTest.java
@@ -10,4 +10,10 @@ public class HentaicafeRipperTest extends RippersTest {
         HentaiCafeRipper ripper = new HentaiCafeRipper(new URL("https://hentai.cafe/kikuta-the-oni-in-the-room/"));
         testRipper(ripper);
     }
+    // This album has a line break (<br />) in the url. Test it to make sure ripme can handle these invalid urls
+    public void testAlbumWithInvalidChars() throws IOException {
+        HentaiCafeRipper ripper = new HentaiCafeRipper(new URL("https://hentai.cafe/chobipero-club/"));
+        testRipper(ripper);
+
+    }
 }

